pkg/cache/types/types.go

```diff
@@ -35,8 +35,8 @@ type ResponseType int
 const (
 	Endpoint ResponseType = iota
 	Cluster
```

Comment:
should `Cluster` sort before `Endpoint`?

Reply:
^ this I have no idea, I had it in the original order and that was causing the bug that @maplebeats was seeing

Reply:
cluster should send before endpoints in cache server, otherwise cds will keep waiting for eds response.